### PR TITLE
clean dep installation step in push_tests

### PR DESCRIPTION
**Pull request by sayakpaul**
# What does this PR do?

As discussed via Slack DM.

### CODE REVIEW

File: .github/workflows/push_tests.yml [hunk 1]
```diff
@@ -21,28 +21,20 @@ env:
...
           fetch-depth: 2
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
       - name: Install dependencies
-        run: |
-          apt-get update && apt-get install libsndfile1-dev libgl1 -y
-          python -m venv /opt/venv && export PATH="/opt/venv/bin:$PATH"
-          python -m uv pip install -e [quality,test]
```
Comment (DN6) [inline .github/workflows/push_tests.yml:39]: I think we still need to install diffusers for the fetching script to work
https://github.com/huggingface/diffusers/blob/ba381ec26b72f1ff33114c692d03121bdb54dc0c/utils/fetch_torch_cuda_pipeline_test_matrix.py#L9

Comment (sayakpaul) [reply .github/workflows/push_tests.yml:39]: Yeah. Let me fix that.

**Review by DN6 — APPROVED** [review]
👍🏽

**Comment by sayakpaul** [comment]
Failing tests are unrelated.

**Pull request merged**